pytestqt/_tests/test_wait_signal.py

```diff
@@ -239,3 +239,30 @@ class TestException(Exception):
     with pytest.raises(TestException):
         with func(arg, timeout=10, raising=raising):
             raise TestException
+
+
+@pytest.mark.parametrize('multiple, do_timeout',
```

Comment:
I just followed the style of `test_wait_signals_handles_exceptions`, but I'll change it for both :smile: